graphql_compiler/tests/test_ir_generation.py

```diff
@@ -6259,3 +6259,7 @@ def test_recursive_field_type_is_subtype_of_parent_field(self):
         }
 
         check_test_data(self, test_data, expected_blocks, expected_location_types)
+
+    def test_animal_born_at_traversal(self):
+        """Ensure that sql composite key traversals work."""
+        # skipping ir generation testing since this is a sql-specific test
```

Comment:
Do you mind adding the expected blocks anyway? This feels like subverting the "all common test data inputs have matching IR and code generation tests" decision we made, and which is also there to help us debug code generation issues because we immediately know whether the issue is due to changes in IR generation or after it.